test/derivative_operators_interface.jl

```diff
@@ -125,7 +131,7 @@ end
     @test Array(A) ≈ correct
     @test sparse(A) ≈ correct
     @test BandedMatrix(A) ≈ correct
-    @test A*y ≈ Array(A)*y
+    @test A*y ≈ Array(A)*y atol=0.1
```

Comment:
why was this added and what does this have to do with upwinding?

Reply:
Fixed after #145.